airflow/utils/email.py

```diff
@@ -226,9 +228,9 @@ def send_mime_email(
             if smtp_starttls:
                 conn.starttls()
             if smtp_user and smtp_password:
-                conn.login(smtp_user, smtp_password)
+                conn.login(smtp_user if smtp_user else "", smtp_password)
```

Comment:
```suggestion
                conn.login(smtp_user, smtp_password)
```

Not needed, this is already inside an` if smtp_user` block.

Reply:
Yeah. Strange why I changed it.